src/iiif/tests.py

```diff
@@ -367,3 +368,85 @@ def test_create_wabo_url(self):
 
         wabo_url = create_wabo_url(metadata=metadata, url_info=url_info)
         self.assertEqual(wabo_url, '2/wabo:SDZ-UIT-COH-628547.PDF/full/1000,1000/0/default.jpg')
+
+    def test_create_cantaloupe_url_and_headers(self):
+        metadata = {
+            'documenten': [
+                {
+                    'barcode': '628547',
+                    'bestanden': [{"filename": "SDZ/UIT/COH/628547.PDF"}]
+                }
+            ]
+        }
+
+        # pre-wabo with no headers
+        url, headers = create_cantaloupe_url_and_headers(
+            {},
+            {'source': 'edepot'},
+            PRE_WABO_IMG_URL,
+            metadata
+        )
+        self.assertEqual(url, PRE_WABO_IMG_URL)
+        self.assertEqual(headers, {})
+
+        # pre-wabo with one header (which we expect to not be used)
+        url, headers = create_cantaloupe_url_and_headers(
```

Comment:
Er zitten nu verschillende cases die getest worden in 1 functie.
Ik denk dat dit mooier is om in losse functies te splitsen.

Reply:
Goed punt. Ik pas dat later aan.